portfolio/src/main/webapp/index.html

```diff
@@ -16,8 +16,8 @@ <h1>Juan Gonzalez's Portfolio</h1>
       <button onclick="addRandomGreeting()">Hello</button>
       <div id="greeting-container"></div>
 
-      <p>Click here to recieve my personalized greeting using fetch():</p>
-      <button onclick="addMyGreeting()">Hello, but for Juan</button>
+      <p>Here is some comment data!:</p>
+      <!-- <button onclick="addMyGreeting()">/data</button> -->
       <div id="my-greeting-container"></div>
```

Comment:
```suggestion
      <div id="comments-container"></div>
```
nit

Reply:
This is changed in a later commit and is now "my-comments-container."